lib/workers/repository/updates/__snapshots__/generate.spec.ts.snap

```diff
@@ -102,7 +102,7 @@ exports[`workers/repository/updates/generate generateBranchConfig() handles @typ
   "prBodyColumns": [],
   "prTitle": "some-title",
   "prettyDepType": "dependency",
-  "recreateClosed": false,
```

Comment:
we probably need to message the old value inside generate function, so it's still available for. templates by users.

Reply:
Working on it.

Reply:
Finished?

Reply:
No, sorry. I am actually lost on this one. Could you help me here plss?